vignettes/introduction.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "An Introduction to sgdnet"
+author: "Johan Larsson"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+bibliography: "sgdnet.bib"
+vignette: >
+  %\VignetteIndexEntry{An Introduction to sgdnet}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Background and motivation
+
+**sgdnet** was developed as a Google Summer of Code 2018. Its goal was to be
```

Comment:
I think you can leave GSoC mentions until the end as a funding acknowledgement. I expect you'll keep maintaining and improving this past the end of GSoC-2018

Reply:
Yup!